magicctapipe/image/tests/test_calib.py

```diff
@@ -0,0 +1,352 @@
+            camera_geoms[tel_id]= telescope.camera.geometry        
+       
+        config_lst = config_calib["LST"]
+
+        extractor_type_lst = config_lst["image_extractor"].pop("type")
+        config_extractor_lst = {extractor_type_lst: config_lst["image_extractor"]}
+
+        calibrator_lst = CameraCalibrator(
+            image_extractor_type=extractor_type_lst,
+            config=Config(config_extractor_lst),
+            subarray=subarray,
+        )
+        
+        LST_bool=True
+        for event in event_source:
+            if (event.count <200) and (tel_id_LST in event.trigger.tels_with_trigger):
+                signal_pixels, image, peak_time = calibrate(
+                    event=event, 
+                    tel_id=tel_id_LST, 
+                    obs_id=obs_id, 
+                    config=config_lst, 
+                    camera_geoms=camera_geoms, 
+                    calibrator=calibrator_lst, 
+                    LST_bool=LST_bool
+                )
+
+                assert len(signal_pixels)==1855
+                assert signal_pixels.dtype==bool
+                assert len(image)==1855
+                assert len(peak_time)==1855
+
+        config_lst["image_extractor"]["type"]=extractor_type_lst  
+
+
+def test_calibrate_MAGIC(dl0_gamma, config_calib, tel_id_MAGIC):
+    
+    assigned_tel_ids = [1,2,3]
+    for input_file in dl0_gamma:
+        event_source = EventSource(
+            input_file,
+            allowed_tels=assigned_tel_ids,
+            focal_length_choice='effective'
+        )
+
+
+        
+        subarray = event_source.subarray    
+
+            
+        tel_descriptions = subarray.tel   
+        camera_geoms={}
+
+        for tel_id, telescope in tel_descriptions.items():
+            camera_geoms[tel_id]= telescope.camera.geometry        
+        
+       
+        config_magic = config_calib["MAGIC"]
+        config_magic["magic_clean"].update({"find_hotpixels": False})
+
+        extractor_type_magic = config_magic["image_extractor"].pop("type")
+        config_extractor_magic = {extractor_type_magic: config_magic["image_extractor"]}
+        magic_clean = {}
+        for k in [1,2]:
+                  
+            magic_clean[k] = MAGICClean(camera_geoms[k], config_magic["magic_clean"])
+        calibrator_magic = CameraCalibrator(
+            image_extractor_type=extractor_type_magic,
+            config=Config(config_extractor_magic),
+            subarray=subarray,
+        )
+        LST_bool=False
+        for event in event_source:
+            if (event.count <200) and (tel_id_MAGIC in event.trigger.tels_with_trigger):
+                signal_pixels, image, peak_time = calibrate(
+                    event=event, 
+                    tel_id=tel_id_MAGIC, 
+                    config=config_magic, 
+                    magic_clean=magic_clean, 
+                    calibrator=calibrator_magic, 
+                    LST_bool=LST_bool
```

Comment:
similarly like @gabemery wrote in the comment above you can write LST_bool=False here and remove line 113 before the loop